PHPUnit/Extensions/Selenium2TestCase.php

```diff
@@ -199,7 +198,7 @@ abstract class Selenium2TestCase extends TestCase
     public static function shareSession($shareSession)
     {
         if (!is_bool($shareSession)) {
-            throw new InvalidArgumentException("The shared session support can only be switched on or off.");
+            throw new \InvalidArgumentException("The shared session support can only be switched on or off.");
```

Comment:
Remove \

Reply:
Then i must alias this exception, because phpunit exception have the same name `PHPUnit\Framework\InvalidArgumentException`